packages/backend/src/apps/gathersg/actions/tag-or-untag-case/index.ts

```diff
@@ -0,0 +1,102 @@
+import type { IRawAction } from '@plumber/types'
+
+import { ZodError } from 'zod'
+import { fromZodError } from 'zod-validation-error'
+
+import StepError, { GenericSolution } from '@/errors/step'
+
+import { GatherSGError } from '../../common/types'
+
+import { requestSchema, responseSchema } from './schema'
+
+const action: IRawAction = {
+  name: 'Tag/Untag case',
+  key: 'tagOrUntagCase',
+  description: 'Tag or untag a case based on the case id (UUID)',
+  arguments: [
+    {
+      label: 'Case ID',
+      key: 'caseId',
+      type: 'string' as const,
+      description:
+        'You can only select a step variable here. You should be using a Tile to store your case IDs to make reference to.',
+      required: true,
+      variables: true,
+      singleVariableSelection: true,
```

Comment:
hmmm no need bah

Reply:
but like no choice leh because i dont want the getDynamicData to be called unnecessarily on every keystroke which is quite costly (same as previous pr)